db/schema.rb

```diff
@@ -0,0 +1,91 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_09_22_131731) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "add_post_ref_to_comments", force: :cascade do |t|
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_add_post_ref_to_comments_on_post_id"
+  end
+
+  create_table "add_post_ref_to_likes", force: :cascade do |t|
+    t.bigint "post_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["post_id"], name: "index_add_post_ref_to_likes_on_post_id"
+  end
+
+  create_table "add_user_ref_to_comments", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_add_user_ref_to_comments_on_user_id"
+  end
+
+  create_table "add_user_ref_to_likes", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_add_user_ref_to_likes_on_user_id"
+  end
+
+  create_table "add_user_ref_to_posts", force: :cascade do |t|
+    t.bigint "user_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_add_user_ref_to_posts_on_user_id"
+  end
```

Comment:
- [ ] These tables are actually not needed so I would suggest removing them just to keep your app clean, also the models that were created with them.

Reply:
@smunozmo thank you Sir very much.